### ISSUE TITLE
Getting Started is difficult to follow

### ISSUE THREAD

**Issue by MaRKHeclim** [MicrosoftDocs/powerquery-docs#437]
I find this guide and the Hello World example hard to follow, and at least partially inaccurate. After a couple days experimenting and combing the documentation I think I have it mostly figured out. 

When I create a new extension project, the bin directory is not created, nor the .mez. 

Sometimes, the "Extension" functions come in as unrecognized, in one of my tests I somehow managed to end up with this not being the case, upon examination, I found an extra line in the settings.json file: "powerquery.general.mode": "SDK" that seems to fix that, I'm not sure if this is a bug that should be reported on the SDK github, or an optional thing that should be noted here. 

Test.Contents in Test.query.pq is always underlined because it cannot find the name, is this normal? If so, it should be called out. 

I have repeatedly had issues setting the credential, some screenshots of the process would be helpful. What would cause it to not find the DataSource Kind? 

How much of this document is first time setup, and how much needs to be done with every change? Is this different for changes to Test.pq and Test.query.pq? 

It would be nice if somewhere that building the extension/.mez (still not sure what the difference is) is mentioned that there were instructions on the correct way to do that. Do I just hit Evaluate current file, or Run TestConnection function, or am I supposed to do something through the menus?

Several things mention the article on installing the SDK, but the table of contents doesn't have such an article. I would think that the installation instructions should be separate from the overview, and the links talking about following the instructions on the installation page to create a project should probably be pointed to the Quick Starts -> Using the Power Query SDK page instead of the overview. 

---
#### Document Details

⚠ *Do not edit this section. It is required for learn.microsoft.com ➟ GitHub issue linking.*

* ID: d8894707-356e-6c86-76fd-c8d6d6c199b6
* Version Independent ID: a4ab5009-5153-df99-e515-8c4c2e4949d5
* Content: [Power Query SDK overview - Power Query](https://learn.microsoft.com/en-us/power-query/install-sdk)
* Content Source: [powerquery-docs/install-sdk.md](https://github.com/MicrosoftDocs/powerquery-docs/blob/main/powerquery-docs/install-sdk.md)
* Service: **powerquery**
* GitHub Login: @ptyx507x
* Microsoft Alias: **miescobar**

**Comment by DougKlopfenstein** [comment]
Hi @MaRKHeclim 
Thanks for your dedication to our documentation. Unfortunately, at this time we have been unable to review your issue in a timely manner and we sincerely apologize for the delayed response. We are closing this issue for now because once the new Power Query SDK using Visual Studio Code reaches general availability, much of the SDK documentation will be reevaluated and rewritten. However, if you feel that it's still a concern, please respond and let us know. If you determine another possible update to our documentation, please don't hesitate to reach out again. #please-close
